tox.ini

```diff
@@ -6,7 +6,8 @@ skip_missing_interpreters = True
 
 commands =
     python --version
-    py.test --pyargs ihatemoney.tests.tests
+    py.test --pyargs ihatemoney.tests.general_test ihatemoney.tests.api_test \
+    ihatemoney.tests.budget_test ihatemoney.tests.history_test
```

Comment:
You should be able to remove all named tests, and just use `py.test --pyargs ihatemoney.tests`. See https://docs.pytest.org/en/latest/goodpractices.html#test-discovery.

Reply:
You're right, i though i've tested that thanks :+1: